common/views/components/IIIFViewer/README.md

```diff
@@ -0,0 +1,4 @@
+## Purpose
+To enable viewing and paging through multi-page IIIF works.
```

Comment:
Since we've had the discussions is it worth noting that this is essentially a static component that takes parameters from the URL, which is controlled by Next rather than having any control itself?